docs/working/examples/example_1.md

```diff
@@ -0,0 +1 @@
+# Example 1
```

Comment:
We should  watch these 'examples' carefully. Ideally, an example is part of a 'user story' or use case that is in the user guide. But might be clearer once the first example or two is created to see if it needs its own 'examples' section or is really part of a the user guide.